src/server/helpers.py

```diff
@@ -195,6 +199,15 @@ def accentless_sort(value):
     return sorted(value, key=lambda i: strip_accents(i[1]).lower())
 
 
+def get_file_date_info(file, type):
+    timestamps_config = get_timestamps_config()
+    value = timestamps_config.get(file, {}).get(type)
+    # Default Published and Last Updated to today
+    if value is None and (type == "published_date" or type == "last_updated_date"):
+        value = datetime.datetime.utcnow().isoformat()
+    return value
```

Comment:
Is this the same thing?

```suggestion
    # Default Published and Last Updated to today
    today = datetime.datetime.utcnow().isoformat()
    return timestamps_config.get(file, {}).get(type, today)
```

Reply:
Seems to work - thanks!

Reply:
Needed a slight varient:

```
    # Default Published and Last Updated to today
    today = datetime.datetime.utcnow().isoformat()
    if type == 'hash':
        return timestamps_config.get(file, {}).get(type)
    else:
        return timestamps_config.get(file, {}).get(type, today)
```